JS/Lesson34/goksel/recipe-app/src/modules/RecipeDetail/recipeDetail.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import "./recipeDetail.css";
+
+export const RecipeDetail = ({ recipeId, recipeImg }) => {
```

Comment:
You don't need to pass recipeImg to this component, instead use image that you get in the response when fetching recipe details by recipeId here.
No need to fix in this PR, fix it in the PR for the assignment 37.